presto-tests/src/test/java/io/prestosql/tests/TestInformationSchemaConnector.java

```diff
@@ -52,12 +52,12 @@
     public void testBasic()
```

Comment:
can you please add tests for `role_grants` here as well?

Reply:
There are 0 tests for any other role or access related stuff in there. I assume it's because it does not work in all connectors.

Reply:
> it's because it does not work in all connectors

This test is using dedicated testing connector called `CountingMockConnector`

> There are 0 tests for any other role or access related stuff

That is a mistake (we didn't catch that). I think having such tests bring a lot of value. Could you please add them? Adding tests for other tables could be done in separate commits.

Reply:
I added some tests....